### PR TITLE
ucrtbase2019: new verb

### PR DESCRIPTION
**Pull request by Gcenx**
I remember a number of online games require this or they get un-synced.

GE-Proton bundled protonfixes uses a custom version of vcrun2019 that uses this installer as it's the final version that still bundled ucrtbase

### CODE REVIEW

File: src/winetricks [hunk 1]
```diff
@@ -12968,7 +12968,7 @@ w_metadata vcrun2017 dlls \
     publisher="Microsoft" \
     year="2017" \
     media="download" \
-    conflicts="vcrun2015 vcrun2019" \
+    conflicts="vcrun2017 vcrun2019 ucrtbase2019" \
```
Comment (austin987) [inline src/winetricks:12971]: Why did you remove vcrun2015, and why was vcrun2017 added? There's no need for it to conflict with itself.

Comment (Gcenx) [reply src/winetricks:12971]: My mistake I'll fix that
